wiki/Interface/en.md

```diff
@@ -1,227 +1,297 @@
+3.  Change Avatar: Open up the edit avatar page in your web browser
+4.  Close : Close this window
+
+## Play Menu
+
+![Play Menu](/wiki/Interface/img/Introscreen2.png)
 
 -   Click Solo (P) to play alone.
--   Click Multi (M) to play with other people. Yow will be directed to Multi Screen.
--   Click Back to return to Main Menu
+-   Click Multi (M) to play with other people. Yow will be directed to the Multi Lobby (see below).
+-   Click Back to return to the main menu.
+
+## Multi Lobby
 
-==Multi's lobby==
+*Full explanation: [Multiplayer](/wiki/Multi)*
 
-*Full explanation: [Multiplayer](Multiplayer "wikilink")*
+![Multi Lobby](/wiki/Interface/img/Multi-Play-2016.png)
+![Multi Host](/wiki/Interface/img/Multi-play!_Host.jpg)
 
-<img src="Multi-Play-2016.png" title="fig:Multi-Play-2016.png" alt="Multi-Play-2016.png" width="280" /> <img src="Multi play! (Host).jpg" title="fig:Multi play! (Host).jpg" alt="Multi play! (Host).jpg" width="280" />
+1. Your rank in the match. This is also shown next to your name.
+2. Your profile information. 
+3. The jukebox. 
+4. Player list - displays player names, their rank (host or player), their mods activated (if any, see \#7), their osu! ranking, and their team (if applicable). 
+5. The name of the match and the password settings.
+6. The beatmap selected. It shows the beatmap as it would in the solo song selection screen. 
+7. The mods that you have activated (see #12), as well as the option to select them. The option marked "Free Mods" toggles whether or not players can select their own mods. If yes, they can pick any combination of mods *except for speed-altering mods like Double Time*. If no, the host decides what mods will be used. The host can pick speed-altering mods regardless of whether or not Free Mods is turned on.
+8. The team mode and win conditions. 
+9. The ready button.
+10. The chat box.
+11. The leave button. 
+12. Where your activated mods appear.
 
-Song Selection Screen
----------------------
+## Song Selection Screen
 
-<img src="soloplay1.jpeg" title="Song selection" alt="Song selection" width="300" />
+![Song Selection](/wiki/Interface/img/Soloplay1.jpeg)
 
-Note: You can identify the song selection for a specific mode by checking the transparent leitmotif.
+Note: You can identify the current mode selected by either looking at the icon in the bottom right, above Mode, or by looking at the transparent icon in the center of the screen.
 
-![](osu.gif "fig:osu.gif")"osu!" is [osu!](Standard "wikilink"), ![](taiko.gif "fig:taiko.gif")"Drum" is [Taiko](Taiko "wikilink"), ![](ctb.gif "fig:ctb.gif")"Apple" is [Catch the Beat](Catch_the_Beat "wikilink") and ![](mania.gif "fig:mania.gif")"Piano" is [osu!mania](Mania "wikilink")
+![][Osu!]"osu!" is [osu!](Standard "wikilink"), ![][Taiko]"Drum" is [Taiko](Taiko "wikilink"), ![][Ctb]"Apple" is [Catch the Beat](Catch_the_Beat "wikilink") and ![][Mania]"Piano" is [osu!mania](Mania "wikilink")
```

Comment:
Can you elaborate? The icons show up just fine in the rich diffs view and on my fork.